src/TurbulenceClosures/turbulence_closure_implementations/anisotropic_minimum_dissipation.jl

```diff
@@ -49,25 +49,32 @@ Arguments
 
 Keyword arguments
 =================
-* `C`: Poincaré constant for both eddy viscosity and eddy diffusivities. `C` is overridden
+* `C`: Poincaré constant for both eddy viscosity and eddy diffusivities. `C` is overridden
        for eddy viscosity or eddy diffusivity if `Cν` or `Cκ` are set, respecitvely.
 
-* `Cν`: Poincaré constant for momentum eddy viscosity.
+* `Cν`: Poincaré constant for momentum eddy viscosity.
 
-* `Cκ`: Poincaré constant for tracer eddy diffusivities. If one number or function, the same
+* `Cκ`: Poincaré constant for tracer eddy diffusivities. If one number or function, the same
         number or function is applied to all tracers. If a `NamedTuple`, it must possess
-        a field specifying the Poncaré constant for every tracer.
+        a field specifying the Poncaré constant for every tracer.
 
 * `Cb`: Buoyancy modification multiplier (`Cb = nothing` turns it off, `Cb = 1` was used by [Abkar16](@citet)).
         *Note*: that we _do not_ subtract the horizontally-average component before computing this
         buoyancy modification term. This implementation differs from [Abkar16](@citet)'s proposal
         and the impact of this approximation has not been tested or validated.
 
-By default: `C = Cν = Cκ = 1/12`, which is appropriate for a finite-volume method employing a
-second-order advection scheme, and `Cb = nothing`, which turns off the buoyancy modification term.
+By default: `C = Cν = Cκ = 1/3`, and `Cb = nothing`, which turns off the buoyancy modification term.
 
 `Cν` or `Cκ` may be numbers, or functions of `x, y, z`.
 
+The default Poincaré constant is appropriate for second order centered advection and is derived by
+descritising the sub grid scale energy production [Verstappen14](@citet). It is 4 times smaller than the 
+staight forward descritisation (which is appropriate for spectral methods) due to the "effective resolution" 
+of second order centered advection schemes.
+
+Discussion demonstraiting the correct energy dissipation behaviour with this Poincaré constant value
+can be found seen in https://github.com/CliMA/Oceananigans.jl/issues/4367.
```

Comment:
```suggestion
By default: `C = Cν = Cκ = 1/3`, and `Cb = nothing`, which turns off the buoyancy modification term.
The default Poincaré constant `C = 1/3` is derived by discretizing subgrid scale energy production [Verstappen14](@citet) assuming a second-order advection scheme. `Cν` or `Cκ` may be numbers, or functions of `x, y, z`.

For more information about the default see the discussion at https://github.com/CliMA/Oceananigans.jl/issues/4367.
```

Reply:
Isn't Verstappen 2014 about a different model? Not AMD? Here's a link to the paper: https://authors.library.caltech.edu/records/dds0a-cgz43/files/06_verstappen.pdf?download=1

I couldn't find "C=1/3" written anywhere in the paper. I guess finding C=1/3 needs some interpretation. Another confusing thing is that they use a different definition of "Poincare constant" (which includes the grid scale) than we do.

Reply:
Ok so the logic is "after equation 14 Verstappen (2014) states that C_finite_volume = 3 x C_spectral", and "C_spectral = 1/12", therefore $C_finite_volume = 1/3".

In my opinion the empirical results are much stronger than these arcane numerology arguments! But, we have the empirical results too so that's good.

Reply:
I really do think we have to write this down. Otherwise the reference is more for show than an explicit argumentation. We have to mention how the argument applies to all constants in SGS models including Smagorinsky, AMD, and QR (it's admittedly been a long time since I read Rozema et al. 2015 but this is a new concept to me)

Reply:
And they also empirically demonstrated it so I said that + that we did it in the issue too